src/preview.ts

```diff
@@ -1,5 +1,5 @@
-import type { DecoratorFunction } from "@storybook/types";
-import { useParameter } from "@storybook/preview-api";
+import type { DecoratorFunction } from "storybook/internal/types";
+import { useParameter } from "storybook/preview-api";
```

Comment:
Related to this.
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#core-changes-and-removals